python/lsst/meas/extensions/ngmix/LMSimpleShape.py

```diff
@@ -93,6 +93,14 @@ def getModelInfo(self, model):
     def __init__(self, config, name, schema, metadata):
         SingleFramePlugin.__init__(self, config, name, schema, metadata)
 
+        self.momentsFailureKey = schema.addField(name + "_momentsFailureFlag", type="Flag", doc="compute Moments failed", units="pixel")
```

Comment:
Also, instead of using "_" explicitly, please use e.g. `schema.join(name, "momentsFailureFlag")`.